build_release.sh

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env sh
+
+set -ue
+
+RELEASE_TAG=${RELEASE_TAG:-$(git describe --tags 2>/dev/null || echo dev)}
+
+BIN_TAG=${BIN_TAG:-$(uname -s)_$(uname -m)}
+
+CUSTOM_LINKING_CATALA_Z3="\
+(-cclib -static
+ -cclib -no-pie
+ -noautolink
+ -cclib -L/home/ocaml/.opam/z3/lib/stublibs
+ -cclib -lz3ml-static
+ -cclib -lz3
+ -cclib -lstdc++
+ -cclib -lthreadsnat
+ -cclib -lzarith
+ -cclib -lgmp
+ -cclib -lcamlstr
```

Comment:
So you've managed to do static linking with Z3 ? How painful was it ?

Reply:
Actually not  harder than [passing the correct configure flag](https://gitlab.inria.fr/lgesbert/catala-ci-images/-/blob/z3static/Dockerfile#L9)

(and then tweaking the link to `-lz3` instead of `-lz3static`)

Reply:
Ha ha ok this was beyond my abilities, I suffered for days on this  thing before giving up.